src/NServiceBus.Testing.Fakes/TestingLoggerFactory.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace NServiceBus.Testing
+{
+    using System;
+    using System.IO;
+    using Logging;
+
+    /// <summary>
+    /// Logger factory which allows to log to a text writer.
+    /// </summary>
+    public class TestingLoggerFactory : LoggingFactoryDefinition
+    {
+        /// <summary>
+        /// Creates a new instance of a testing logger factory.
+        /// </summary>
+        public TestingLoggerFactory()
+        {
+            level = new Lazy<LogLevel>(() => LogLevel.Debug);
+            writer = new Lazy<TextWriter>(() => new StringWriter());
```

Comment:
I thought about that too.I should have expressed it properly in the PR.

`StringWriter` this switches a flag and calls base.Dispose
http://referencesource.microsoft.com/#mscorlib/system/io/stringwriter.cs,74

base.Dispose does nothing.

http://referencesource.microsoft.com/#mscorlib/system/io/textwriter.cs,135

So I'd say no problem. I could also switch to https://msdn.microsoft.com/en-us/library/system.io.textwriter.null(v=vs.110).aspx

Thoughts?